cmd/generate-config/main.go

```diff
@@ -11,74 +11,42 @@ import (
 )
 
 func main() {
-	defaultsForCI := flag.Bool("ci", false, "sane defaults for CI testing")
+	defaultsForCI := flag.Bool("ci", false, "Populate the configuration with sane defaults for use in CI")
 	serverName := flag.String("server", "", "The domain name of the server if not 'localhost'")
 	dbURI := flag.String("db", "", "The DB URI to use for all components if not SQLite files")
+	normalise := flag.String("normalise", "", "Normalise a given configuration file")
```

Comment:
Unclear what this is supposed to do from the docstring alone.

Reply:
May your humble guinea pig suggest that what you want to do is to: "update" an existing config file to the latest and greatest, adding missing defaults, rather than "normalizing" things?
But then, any term is good.

Reply:
I specifically didn't want to use words like "update" because it isn't really what it does — it won't migrate an old config version to a new one for example, and it won't correct the formatting of old options vs new ones at present.